openedx/features/ucsd_features/ecommerce/utils.py

```diff
@@ -12,6 +12,19 @@
 
 
 def is_user_eligible_for_discount(request, course_key):
+    """
+    Checks if the user in the request is eligible to get a discount voucher for course with course_key.
+    A user is eligible for discount iff:
```

Comment:
```suggestion
    A user is eligible for discount if:
```

Reply:
This is intentional. "Iff" = "If and only if"